### PR TITLE
Enable sp swift func tests

### PR DESCRIPTION
**Pull request by cazino**
This make the swift functional tests related to SP being run instead of being skipped.

### CODE REVIEW

File: jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh [hunk 1]
```diff
@@ -23,6 +23,35 @@ function enable_sproxyd_driver {
...
+sproxyd_endpoints = http://127.0.0.1:81/proxy/arc
+
+[storage-policy:1]
+read = paris-arc
+write = paris-arc
+
+EOF
+}
+
+function sylimk_ring_files {
```
Comment (JordanP) [inline jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh]: typo : symlink

File: jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh [hunk 2]
```diff
@@ -23,6 +23,35 @@ function enable_sproxyd_driver {
...
+    ln -s ${SWIFT_CONF_DIR}/object.ring.gz ${SWIFT_CONF_DIR}/object-1.ring.gz
+    ln -s ${SWIFT_CONF_DIR}/container.ring.gz ${SWIFT_CONF_DIR}/container-1.ring.gz
```
Comment (JordanP) [inline jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh]: Why do you have to create another container ring ? 

This documentation here : http://docs.openstack.org/developer/swift/development_saio.html (take a look at the remakerings script) create one additional objects ring for each new policy. But there's one/1 container ring in total.

File: jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh [hunk 3]
```diff
@@ -23,6 +23,34 @@ function enable_sproxyd_driver {
...
+}
+
+function symlink_ring_files {
+    ln -s ${SWIFT_CONF_DIR}/object.ring.gz ${SWIFT_CONF_DIR}/object-1.ring.gz
+}
+
+function enable_storage_policies {
+    amend_swift_conf
+    create_storage_policies_conf
+    sylimk_ring_files
```
Comment (JordanP) [inline jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh]: Typo corrector here: we have a typo !

**Comment by cazino** [comment]
Functional tests results : https://37.187.159.67:5443/job/swift-functional-tests/151/

**Comment by JordanP** [comment]
LGTM

**Pull request merged**